modules/moa-faq-installing.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// getting_started_moa/moa-faq.adoc
+
+
+[id="moa-faq-installing"]
+
+= Installation troubleshooting
+
+== Inspect install or uninstall logs
+
+Run the following command to generate install logs. Replace <my_cluster_name> with the name of your cluster.
+
+[source,terminal]
+----
+$ moactl logs install <my_cluster_name>
+----
+
+Run the following command to generate uninstall logs. Replace <my_cluster_name> with the name of your cluster.
+
+[source,terminal]
+----
+$ moactl logs uninstall <my_cluster_name>
+----
+
+== Verify your AWS account does not have an SCP
+
+Run the following command to verify your AWS account has the correct permissions.
+
+[source,terminal]
+----
+$ moactl verify permissions
+----
+
+If you receive any errors, double check to ensure than an link:https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_type-auth.html#orgs_manage_policies_scp[SCP] is not applied to your AWS account. If you are required to use an SCP, see link:https://www.openshift.com/dedicated/ccs#scp[Red Hat Requirements for Customer Cloud Subscriptions] for details on the minimum required SCP.
+
+== Verify your AWS account and quota
+
+Run the following command to verify you have the available quota on your AWS account. 
+
+[source,terminal]
+----
+$ moactl verify quota
+----
+
+If you need to increase your quota, navigate to your link:https://aws.amazon.com/console/[AWS console], and request a quota increase for the service that failed.
```

Comment:
AWS quota requests are typically tied to a specific region.  Make sure you are requesting quota for the region you intend to use for installation.